cmd/keymaster/main.go

```diff
@@ -496,6 +497,10 @@ func main() {
 	flag.Usage = Usage
 	flag.Parse()
 	logger := cmdlogger.New()
+	if *printVersion {
+		fmt.Println(Version)
```

Comment:
nit: use the logger instead of directly using fmt.

Reply:
the thing is that logger prints it out to stderr, it made more sense to me to print it to stdout